tests/unit/helpers.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2020 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+import os
+import tempfile
+from typing import Tuple
+
+
+class Sandbox:
```

Comment:
This name is not very descriptive of its use. Consider switching to `TempFolderSandbox` or something else that's more descriptive.